src/Stratis.Features.FederatedPeg/TargetChain/WithdrawalTransactionBuilder.cs

```diff
@@ -6,11 +6,19 @@
 using Stratis.Bitcoin.Features.Wallet;
 using Stratis.Features.FederatedPeg.Interfaces;
 using Stratis.Features.FederatedPeg.Wallet;
+using Stratis.SmartContracts.Core.State;
 using Recipient = Stratis.Features.FederatedPeg.Wallet.Recipient;
 using TransactionBuildContext = Stratis.Features.FederatedPeg.Wallet.TransactionBuildContext;
 
 namespace Stratis.Features.FederatedPeg.TargetChain
 {
+    public class BuildWithdrawalTransactionResult
+    {
+        public bool Success => this.Transaction != null;
+        public bool Reject { get; set; }
```

Comment:
Nice. Could you put a comment on this property `Rejected` that explains what it's for

Reply:
Done.